core/db/entities.py

```diff
@@ -41,7 +41,8 @@ class Submission(BaseModel):
 
     __tablename__ = "submission_dim"
 
-    submission_id = sqla.Column(sqla.String(), nullable=False, unique=True)
+    # TODO: Change submission ID from an int value to a string that includes programme or fund type?
```

Comment:
can remove this todo now?

Reply:
old rebase was messing with the branch - should be fixed now